src/web/js/trove/chart-lib.js

```diff
@@ -174,6 +214,116 @@
 
   //////////////////////////////////////////////////////////////////////////////
 
+  /**
```

Comment:
It's probably worth saying up top that this was copied from a jsfiddle more explicitly.

Reply:
The jsfiddle is actually from @eyanje so I'm not sure it's copied, probably more of an example. But it still might be worth putting up top for reference! Do you want me to do that in this PR or do you want to do it directly on horizon since this has already been merged. Sorry not 100% up to date on PR etiquette.